people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Developers
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
+- Wes McKinney
+- Wouter Overmeire
```

Comment:
It's addressed in the governance doc -- basically I have to reach out to inactive people to see if they want to re-engage otherwise be removed from the active core team.